packages/techdocs-common/src/stages/publish/googleStorage.ts

```diff
@@ -169,30 +169,23 @@ export class GoogleGCSPublish implements PublisherBase {
       // Files with different extensions (CSS, HTML) need to be served with different headers
       const fileExtension = path.extname(filePath);
       const responseHeaders = getHeadersForFileExtension(fileExtension);
+      res.writeHead(200, {
+        ...{
+          'Transfer-Encoding': 'chunked',
```

Comment:
Would expect this to be automatically written when piping if it is needed, otherwise I assume it just streams the entire body. Is it really needed?

Reply:
A quick test indicates it is not needed.  Nice!